src/main/java/com/xiao/users/config/AppConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.xiao.users.config;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
+public class AppConfig {
+
+  @Value(value = "${info.app.name}")
```

Comment:
no need to use app name for define consumer group

Reply:
this is producer, not comsumer